kn.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: kn
+  version: 1.17.0
+  epoch: 0
+  description: Knative CLI implementation
+  copyright:
+    - license: Apache-2.0
+
```

Comment:
Should we add `provides:` here for `knative=${{package.full-version}}` and `kn=${{package.full-version}}`

Reply:
added a provides for `kn` but not for `knative` mostly thinking that in future maybe we'll have a meta `knative` package someday. 💭